recipes/tcl/8.6.9/conanfile.py

```diff
@@ -0,0 +1,222 @@
+
+    def _build_autotools(self):
+        conf_args = [
+            "--enable-threads",
+            "--enable-shared" if self.options.shared else "--disable-shared",
+            "--enable-symbols" if self.settings.build_type == "Debug" else "--disable-symbols",
+            "--enable-64bit" if self.settings.arch == "x86_64" else "--disable-64bit",
+            "--prefix={}".format(self._install_folder),
+        ]
+        autoTools = self._get_auto_tools()
+        autoTools.configure(configure_dir=self._get_configure_dir(), args=conf_args, vars={"PKG_CFG_ARGS": " ".join(conf_args)})
+
+        # https://core.tcl.tk/tcl/tktview/840660e5a1
+        for root, _, files in os.walk(self.build_folder):
+            if "Makefile" in files:
+                tools.replace_in_file(os.path.join(root, "Makefile"), "-Dstrtod=fixstrtod", "", strict=False)
+
+        with tools.chdir(self.build_folder):
+            autoTools.make()
+
+    def build(self):
+        self._fix_sources()
+        if self.settings.compiler == "Visual Studio":
+            # do not treat warnings as errors 
+            tools.replace_in_file(os.path.join(self._source_subfolder, "win", "rules.vc"), "cwarn = $(cwarn) -WX", "")
+            self._build_nmake()
+        else:
+            self._build_autotools()
+
+    def package(self):
+        if self.settings.compiler == "Visual Studio":
+            self._build_nmake("install")
+        else:
+            with tools.chdir(self.build_folder):
+                autoTools = self._get_auto_tools()
+                autoTools.install()
+                autoTools.make(target="install-private-headers")
+            pkgconfig_dir = os.path.join(self._install_folder, "lib", "pkgconfig")
+            if os.path.isdir(pkgconfig_dir):
+                shutil.rmtree(pkgconfig_dir)
+        self.copy(pattern="license.terms", dst="licenses", src=self._source_subfolder)
+
+        tclConfigShPath = os.path.join(self._install_folder, "lib", "tclConfig.sh")
+        package_path = os.path.join(self._install_folder)
+        if self._is_mingw_windows:
+            package_path = package_path.replace("\\", "/")
+        tools.replace_in_file(tclConfigShPath,
+                              package_path,
+                              "${TCL_ROOT}")
+        tools.replace_in_file(tclConfigShPath,
+                              "\nTCL_BUILD_",
+                              "\n#TCL_BUILD_")
+
+        tools.replace_in_file(tclConfigShPath,
+                              "\nTCL_SRC_DIR",
+                              "\n#TCL_SRC_DIR")
+
+    def package_info(self):
+        libs = []
+        libdirs = []
+        for root, _, _ in os.walk(os.path.join(self._install_folder, "lib"), topdown=False):
+            newlibs = tools.collect_libs(self, root)
+            if newlibs:
+                libs.extend(newlibs)
+                libdirs.append(root)
+        if self.settings.compiler == "Visual Studio":
+            libs.extend(["netapi32"])
```

Comment:
I think these libraries (and below) can be added on Windows (independent of the compiler):
`netapi`, `user32`, `userenv` and `ws2_32`.